dist/backdraft.js

```diff
@@ -1105,16 +1205,43 @@ _.extend(Plugin.factory, {
 
     _onFilterClick: function () {
       $("input[type=text]", this.head).trigger("change");
+      // Update ajaxsource on datatable
+      this._updateAjaxSource();
       this.table.dataTable._fnAjaxUpdate();
       this.$(".toggle-filter-button").popoverMenu('hide');
     },
 
     _onClearClick: function () {
       this.child("filter-menu").clear();
+      this._updateAjaxSource();
       this.table.dataTable._fnAjaxUpdate();
       this.$(".toggle-filter-button").popoverMenu('hide');
     },
 
+    _updateAjaxSource: function() {
+      // get ajax url
+      var ajaxURL = this.parent.dataTable.fnSettings().sAjaxSource;
+      // get the endpoint of ajax url
+      var endpoint = ajaxURL.split("?")[0];
```

Comment:
You split this three times in a row, perhaps saving the split in a local would be a good idea.